themes/digital.gov/layouts/shortcodes/note.html

```diff
@@ -1,129 +1,37 @@
-  {{- if and (.Get 0) (eq (.Get 0) "activity") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use
-          xlink:href="{{ "uswds/img/sprite.svg#assessment" | relURL }}"
-        ></use>
-      </svg>
-      Activity
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "action") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use xlink:href="{{ "uswds/img/sprite.svg#campaign" | relURL }}"></use>
-      </svg>
-      Action
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "alert") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use xlink:href="{{ "uswds/img/sprite.svg#report" | relURL }}"></use>
-      </svg>
-      Alert
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "comment") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use xlink:href="{{ "uswds/img/sprite.svg#comment" | relURL }}"></use>
-      </svg>
-      Comment
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "video") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use
-          xlink:href="{{ "uswds/img/sprite.svg#fast_forward" | relURL }}"
-        ></use>
-      </svg>
-      Video
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "group") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use xlink:href="{{ "uswds/img/sprite.svg#groups" | relURL }}"></use>
-      </svg>
-      Who can join?
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "plain") -}}
-  {{- else -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use
-          xlink:href="{{ "uswds/img/sprite.svg#notifications" | relURL }}"
-        ></use>
-      </svg>
-      Note
-    </h4>
-  {{- end -}}
-
+{{ $variant := (.Get "variant") }}
+{{ $icon := "" }}
+
+{{ if eq $variant "Activity" }}
+  {{ $icon = "assessment" }}
+{{ else if eq $variant "Action" }}
+  {{ $icon = "campaign" }}
+{{ else if eq $variant "Alert" }}
+  {{ $icon = "report" }}
+{{ else if eq $variant "Comment" }}
+  {{ $icon = "comment" }}
+{{ else if eq $variant "Video" }}
+  {{ $icon = "fast_forward" }}
+{{ else if eq $variant "Who can Join?" }}
+  {{ $icon = "groups" }}
+{{ else if eq $variant "Note" }}
+  {{ $icon = "notifications" }}
+{{ else if eq $variant "Disclaimer" }}
+  {{ $icon = "info" }}
+{{ end }}
```

Comment:
If we're adding disclaimer code here, we could call this component in `disclaimer.html` with predefined settings.